metals/src/main/scala/scala/meta/internal/metals/AdjustLspData.scala

```diff
@@ -77,6 +72,24 @@ trait AdjustLspData {
   }
 }
 
+case class AdjustLspPos(
```

Comment:
I normally don't love case classes that are verbs. What do you think of `AdjustedLspPos`?

Reply:
Fixed!